spec/controllers/albums_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe AlbumsController, type: :controller do
+  it_behaves_like "a medium controller" do
+    let(:model) { Album }
+    let (:good_params) do
+      {
+        album: { name: "zzzTest", description: "Album's description", artist: "Album's artist"
+        }
```

Comment:
I think either this closing curly brace should be on the same line as `album: { ...` or the key/value pairs in the `album` sub-hash should be split out onto their own lines, to keep with the style of the containing hash.